app/workers/update_schedule_worker.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateScheduleWorker
+  include Sidekiq::Worker
+
+  def self.perform
+    Student.all.each do |student|
+      student.update(at_school: false)
+      UpdateScheduleJob.perform(student)
```

Comment:
So then how would they be marked absent? If they don't check in to their first class, would it go to false?